src/components/dashboard/main-content/course-enrollments/tests/CourseEnrollments.test.jsx

```diff
@@ -60,28 +58,39 @@ function renderEnrollmentsComponent(initialProps) {
     </Provider>,
   );
 }
+describe('Course enrollements', () => {
+  beforeEach(() => {
+    updateCourseCompleteStatusRequest.mockImplementation(() => {});
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  test('loads enrollments component', () => {
+    renderEnrollmentsComponent(initProps);
+    expect(screen.getByText('My courses in progress')).toBeInTheDocument();
```

Comment:
suggest: again just me being paranoid I think, we __can__ if you want, make these into constants in https://github.com/edx/frontend-app-learner-portal-enterprise/blob/09daa5b1afb92c1add39cb08e6c49e2370706b89/src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx#L148 and reuse them here..

Reply:
😄 I *do* like using constants... But sadly I think this ticket has had it's scope enough for now. Next time I'm in this code, though...

Reply:
np, let's address it in other PRs when we work on this next